src/components/paragraph/Paragraph.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Content from '../content';
+
+const Paragraph = ({ nodeType, content, entryMap, level }) => (
+    <div data-type={ nodeType }>
+        <Content data={ { content } } entryMap={ entryMap } level={ level + 1 } />
```

Comment:
[This](https://github.com/moxystudio/react-contentful-rich-text/pull/1/files#r417415723) applies here.